ccmlib/common.py

```diff
@@ -784,10 +780,7 @@ def parse_settings(args):
         elif val.lower() == "false":
             val = False
         else:
-            try:
-                val = int(val)
-            except ValueError:
-                pass
+            val = yaml.safe_load(str(val))
```

Comment:
> no need to cast `val` to `str`.

this comment is not addressed. could you please reconsider it?

Reply:
removed